### ISSUE TITLE
repository to use for dojo examples

### ISSUE THREAD

**Issue by MHBauer** [Open-I-Beam/DockerDojo#7]
should we use this for our git examples?

**Comment by duglin** [comment]
Trying to setup a fork of Docker under Open-I-Beam - so perhaps we should use that if we can, unless you think that will be harder on folks for some reason.  I do plan on them cloning the DockerDojo repo too but its probably a good thing to keep test issues/PRs on the Docker one so that we can clean it up by just trashing it after the class - keeping DockerDojo relatively clean.  Make sense?

**Comment by MHBauer** [comment]
makes sense from a cleanliness perspective. I was more worried about having 20 people all cloning a hundred meg repo at the same time.

**Comment by duglin** [comment]
If our network can't handle it then it deserves to be DOS'd :-)
